flow/environment.py

```diff
@@ -434,9 +434,6 @@ def add_args(cls, parser):
 
         """
         super().add_args(parser)
-        parser.add_argument(
```

Comment:
Via command line, this is the only gateway to the `walltime` argument in `submit`. I propose of deprecating it and not removing it.

Reply:
I agree, that makes sense.